src/payments/p2wsh.js

```diff
@@ -14,6 +15,14 @@ function stacksEqual(a, b) {
     return x.equals(b[i]);
   });
 }
+function chunkHasUncompressedPubkey(chunk) {
+  if (Buffer.isBuffer(chunk) && chunk.length === 65) {
+    if (ecc.isPoint(chunk)) return true;
+    else return false;
+  } else {
+    return false;
+  }
```

Comment:
I think it could be even simpler.
```suggestion
  if (Buffer.isBuffer(chunk) && chunk.length === 65 && ecc.isPoint(chunk)) {
    return true;
  }
  return false;
```